rules-tests/CodeQuality/Rector/Property/ImproveDoctrineCollectionDocTypeInEntityRector/Source/Training.php

```diff
@@ -4,7 +4,13 @@
 
 namespace Rector\Doctrine\Tests\CodeQuality\Rector\Property\ImproveDoctrineCollectionDocTypeInEntityRector\Source;
 
+use Doctrine\ORM\Mapping as ORM;
+
 final class Training
 {
-
+    /**
+     * @ORM\Column(name="id", type="string")
+     * @ORM\Id
+     */
+    private string $id;
```

Comment:
Looks good 👍 

Could you add test fixture for attribute+ int type?

Reply:
sure, added integer type for annotation and attribute 👍 https://github.com/rectorphp/rector-doctrine/pull/362/commits/f84e8a5dab7a367b41d0d6c248d59f815b99c442

Reply:
also added more fixture for both attribute on current and target class https://github.com/rectorphp/rector-doctrine/pull/362/commits/724422732704f21b4a4fd1c9c218b470415d4cae

Reply:
also added constant fetch read on type target https://github.com/rectorphp/rector-doctrine/pull/362/commits/fc0a97466617a93d73172df2b00c07f890a953c3